{
  "Header": {
    "Original": "CVE-2019-18446 (An issue was discovered in GitLab Community and Enterprise Edition 8.1 ...)",
    "Line": 266077,
    "ID": "CVE-2019-18446",
    "Description": "(An issue was discovered in GitLab Community and Enterprise Edition 8.1 ...)"
  },
  "Annotations": [
    {
      "Original": "[experimental] - gitlab 12.2.9-1",
      "Line": 266078,
      "Type": "package",
      "Release": "experimental",
      "Package": "gitlab",
      "Kind": "fixed",
      "Version": "12.2.9-1"
    },
    {
      "Original": "- gitlab 12.6.8-3",
      "Line": 266079,
      "Type": "package",
      "Package": "gitlab",
      "Kind": "fixed",
      "Version": "12.6.8-3"
    },
    {
      "Original": "NOTE: https://about.gitlab.com/blog/2019/10/30/security-release-gitlab-12-dot-4-dot-1-released/",
      "Line": 266080,
      "Type": "NOTE",
      "Description": "https://about.gitlab.com/blog/2019/10/30/security-release-gitlab-12-dot-4-dot-1-released/"
    }
  ]
}